controls/radtreeview/features/ui-virtualization.md

```diff
@@ -20,49 +20,40 @@ The following tutorial shows how to bind to a collection of business objects and
 
 Here is a simple __RadTreeView__ declaration.

```

Comment:
We can paraphrase to use the fact that the examples are named -> __Example 1__ demonstrates a simple __RadTreeView__ declaration.